### ISSUE TITLE
[A11y] focus loss when dismissing install banner

### ISSUE THREAD

**Issue by dmochak** [ampproject/amphtml#4942]
## Short description of your issue:

Screenreader focus on Android becomes lost after dismissing the AMP install banner.

## How do we reproduce the issue?

Please post a public URL and ideally post a reduced test case. E.g. on jsbin.com that exhibits only your issue and nothing else.

1. Activate TalkBack (settings > accessibility > TalkBack)
2. open the install banner from by clicking the two link.
3. Dismiss the banner by doubletapping the dismiss button.
4. Swipe left or right.

Actual result: TalkBack plays the the sound that indicates it has reached the beginning/end of the swipe order, thus it has nowhere to go. Focus can only be regained by exploring to another part of the page by touch.

Expected result: focus should be restored to the page content after the banner is dismissed.

## What browsers are affected?

Anroid (Chrome)

## Which AMP version is affected?

Powered by AMP ⚡ HTML – Version 1473296795712

**Comment by dmochak** [comment]
Whoops, link to demo: https://amp-mk-3.herokuapp.com/examples/viewer.html

**Comment by ericlindley-g** [comment]
to/ @mkhatib

**Comment by mkhatib** [comment]
What is "Swipe right and swipe left" step is about?

**Comment by ericlindley-g** [comment]
@mkhatib it's part of [gestural navigation](https://support.google.com/accessibility/android/answer/6151827?hl=en) in Talkback. (I had to look it up, too : )

**Comment by mkhatib** [comment]
So I am unable to reproduce this, and @aghassemi has experimented with this as well, it seems the document still gets back the focus and so the user can resume reading the page from start by swiping right.

I am not sure if anything else need to be done here.

**Comment by mkhatib** [comment]
Closing this since we weren't able to repro.
